src/components/TokenLocking/MilesReceipt.tsx

```diff
@@ -25,68 +25,75 @@ const MilesReceipt = ({
 
   return (
     <Modal open={open} onClose={onClose} slotProps={{ backdrop: { sx: { backgroundColor: '#636669BF' } } }}>
-      <div className={css.milesReceipt}>
+      <>
         <div className={clsx(css.gradientBackground, { [css.unlockGradient]: isUnlock })} />
-        <Stack className={css.leftReceipt}>
+        <Stack className={css.milesReceipt}>
           <SvgIcon component={SafeExplorers} inheritViewBox sx={{ width: '282px', height: 'auto' }} />
-          <Stack mt="auto">
-            <Typography variant="h1" component="div" fontWeight="bold" mb={1}>
-              {isUnlock ? 'Your tokens are unlocked' : 'Welcome on board'}
+          <Stack mt="auto" className={css.topReceipt}>
+            <Typography variant="h1" component="div" fontWeight="bold" mb={1} mt={7}>
+              {isUnlock ? 'Your tokens are unlocked' : 'Locking started...'}
             </Typography>
-            <Typography mb={5}>
-              You successfully started {isUnlock ? 'unlocking' : 'locking'} {amount} SAFE. Once the transaction is
-              signed and executed the changes will be reflected in this App.
-              <br />
-              {isUnlock ? 'Withdrawal will be available in 24h' : ''}
+            <Typography mb={4}>
+              You {isUnlock ? 'unlocked' : 'successfully started locking'} {amount} SAFE.
```

Comment:
This is not correct. In both cases the user just started the transaction. It might not be fully executed yet so I'd still say "You successfully started unlocking" here.

Reply:
Same applies to the block underneath.